src/Access/AccessRights.cpp

```diff
@@ -258,7 +258,6 @@ namespace
             case TABLE_LEVEL: return AccessFlags::allFlagsGrantableOnTableLevel();
             case COLUMN_LEVEL: return AccessFlags::allFlagsGrantableOnColumnLevel();
         }
```

Comment:
Maybe consider adding a `chassert`? Because `Level` is an enum, not an enum class, it is "easy" to pass in an invalid value.